jre/guide/java/java_jre_revocation_check/oval/jre.xml

```diff
@@ -0,0 +1,34 @@
+<def-group>
+  <definition class="compliance" id="java_jre_revocation_check" version="1">
+    <metadata>
+      <title>Check All Certificates</title>
```

Comment:
This title needs to be updated to something like `Force Certificate Revocation Checking`

Reply:
The rule title should also match this updated title.